grails-app/conf/application.groovy

```diff
@@ -37,23 +41,23 @@ environments {
         tagURL = "${cavaWpRestUrl}wp-json/wp/v2/tags?per_page=400"
 
         if(System.properties['grails.serverURL']) {
-            grails.serverURL = System.properties['grails.serverURL']
+            server.contextPath = System.properties['grails.serverURL']
         } else {
-            grails.serverURL = "https://api.interactiveoceans.washington.edu/media"
+            server.contextPath = "https://api.interactiveoceans.washington.edu/media"
         }
-        println "serverURL is ${grails.serverURL}"
+        println "server.contextPath is ${server.contextPath}"
     }
     development {
         cavaWpRestUrl = "http://localhost:8888/"
         cavaWpPassword = "iRmQcp@YOCrbRxOHgCT9#4ZT"
```

Comment:
Btw, this is just local WP instance password right?

Reply:
Yes